include/igraph_qsort.h

```diff
@@ -25,6 +25,7 @@
 #define IGRAPH_QSORT_H
 
 #include "igraph_decls.h"
+#include "igraph_types.h"
```

Comment:
Do you remember why this include is necessary here? The only thing that is non-standard in this file seems to be `size_t`, which should be covered by `stddef.h`.

Reply:
The `IGRAPH_UNUSED` definition is in this header. I would be more than happy to move this definition, because it does not really seem to fit in this header, but that would touch a lot of more files then presumably.

Reply:
Got it. We could move it to `igraph_decls.h` -- after all, it _declares_ that a variable is unused :)

Reply:
I am now doing this, I just move the declaration to `igraph_decls.h` and remove the `igraph_types.h` inclusion here. It might be that there are some superfluous includes still on `igraph_types.h`, although that is unlikely.